src/js/d3Utils.js

```diff
@@ -0,0 +1,42 @@
+/*!
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
```

Comment:
I think the copyright statement in all of our js files is wrong. It should be the following:

```javascript
/*
Copyright 2015 OCAD University

Licensed under the Educational Community License (ECL), Version 2.0 or the New
BSD license. You may not use this file except in compliance with one these
Licenses.

You may obtain a copy of the ECL 2.0 License and BSD License at
https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
*/
```

This would match infusion and what is in the [LICENSE.txt](https://github.com/fluid-project/chartAuthoring/blob/master/LICENSE.txt) and [package.json](https://github.com/fluid-project/chartAuthoring/blob/master/package.json#L9-L18). Could you update all the of files to have the correct copyright statement.